### ISSUE TITLE
inconsistency in threshold distances between pysal and geodaspac

### ISSUE THREAD

**Issue by sjsrey** [pysal/pysal#224]
_Original author: sjsrey (October 19, 2012 19:39:02)_

just noticed a slight discrepancy between geodaspace and pysal.
computing a threshold distance in geodaspace for NAT.shp, setting
arc distance to miles, yields: 90.8765250783
in pysal, the same command yields 90.865247381472884.
there is a slight discrepancy, although both are essentially the same
up to a 1/100th of a mile: 90.88 vs 90.87, about 16 meters difference.
basically nothing substantially to worry about, but why would it
be different? presumably both call
pysal.min_threshold_dist_from_shapefile(&quot;NAT.shp&quot;,
       radius=pysal.cg.sphere.RADIUS_EARTH_MILES)
so any rounding errors should be the same. did that function somehow
change and geodaspace is using an old one?
any ideas?
(same NAT.shp, btw)

_Original issue: http://code.google.com/p/pysal/issues/detail?id=224_
